erpnext/config/accounts.py

```diff
@@ -389,6 +389,11 @@ def get_data():
 			"label": _("Settings"),
 			"icon": "fa fa-cog",
 			"items": [
+				{
```

Comment:
@P-Froggy Account Settings is already there under "Accounting Masters" section. Please check line no. 149. Closing this